lessonbuilder/tool/src/webapp/js/show-page.js

```diff
@@ -2974,6 +2974,16 @@ $(document).ready(function() {
 	$('.no-highlight').folderListing({
 		enableHighlight: false,
 	});
+
+	$('a[href*="access/basiclti"]').each(function() {
+		var href = this.href;
+		if ( this.href == null ) return;
```

Comment:
You should use `===` here, unless you're relying on type coercion.

```suggestion
		if ( this.href === null ) return;
```